<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskWindow</class>
 <widget class="QWidget" name="TaskWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButtonSolveChord">
     <property name="text">
      <string>Solve Equation (Chord Method)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonEmbedMessage">
     <property name="text">
      <string>Embed Message in Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonLogout">
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
